cmd/agent/main.go

```diff
@@ -71,7 +94,7 @@ func send(workerID int, adr string, msgs api.MetricsList, key string) (err error
 		}
 		after := (retry+1)*2 - 1
 		time.Sleep(time.Duration(after) * time.Second)
-		log.Printf("worker #%d: %s Retry %d ...", workerID, err.Error(), retry+1)
+		slog.Warn(fmt.Sprintf("worker #%d: %s Retry %d ...", workerID, err.Error(), retry+1))
```

Comment:
Такая строка будет больше в `slog` стиле.
```suggestion
		slog.Warn("Retry attempt", "worker", workerID, "error", err, "retry", retry+1)
```